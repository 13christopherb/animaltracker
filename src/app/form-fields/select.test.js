import React from 'react';
import {Select} from './';
import renderer from 'react-test-renderer'

describe('<Input />', () => {
    const defaultProps = {
        meta: {
            touched: false,
            error: null
        },
        id: 'location',
        name: 'location',
        title: 'Location',
        options: (['NRO', 'MBO', 'SLO'])
    };

    describe('render()', () => {
        test('should render the component normally', () => {
            const component = renderer.create(<Select {...defaultProps} />).toJSON();
            expect(component).toMatchSnapshot()
        });
    });
});