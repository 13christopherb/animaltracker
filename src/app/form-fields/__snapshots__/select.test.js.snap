// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> render() should render the component normally 1`] = `
<div
  className="form-group"
>
  <label>
    Location
  </label>
  <select
    className="custom-select "
    id="location"
    name="location"
  >
    <option
      disabled={true}
      value=""
    />
    <option
      label="NRO"
      value="NRO"
    >
      NRO
    </option>
    <option
      label="MBO"
      value="MBO"
    >
      MBO
    </option>
    <option
      label="SLO"
      value="SLO"
    >
      SLO
    </option>
  </select>
</div>
`;
